### DISCUSSION TITLE
A question about docs.

### DISCUSSION THREAD

**Discussion by elisalimli** [superagent-ai/superagent#628] [General]
https://docs.superagent.sh/overview/getting-started/basic-example

In the second step, we are creating an agent 

the code for creating an agent endpoint (https://github.com/homanp/superagent/blob/main/libs/superagent/app/api/agents.py#L61).

The endpoint to create an agent already creates an agent_llm, why in the third step we're adding agent_llm again?

https://github.com/homanp/superagent/blob/main/libs/superagent/app/api/agents.py#L88C1-L88C9

**Comment by elisalimli** [comment]
why do Agents and LLM table have many-to-many relationship? How can an agent have more than one LLM?

https://github.com/homanp/superagent/blob/main/libs/superagent/prisma/schema.prisma#L98

**Comment by homanp** [comment]
> https://docs.superagent.sh/overview/getting-started/basic-example
> 
> 
> 
> In the second step, we are creating an agent 
> 
> 
> 
> the code for creating an agent endpoint (https://github.com/homanp/superagent/blob/main/libs/superagent/app/api/agents.py#L61).
> 
> 
> 
> The endpoint to create an agent already creates an agent_llm, why in the third step we're adding agent_llm again?
> 
> 
> 
> https://github.com/homanp/superagent/blob/main/libs/superagent/app/api/agents.py#L88C1-L88C9
> 
> 

Just haven't updated the docs After the changes.

**Comment by homanp** [comment]
> why do Agents and LLM table have many-to-many relationship? How can an agent have more than one LLM?
> 
> 
> 
> https://github.com/homanp/superagent/blob/main/libs/superagent/prisma/schema.prisma#L98

Legacy.
